kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaTableWriteSuiteBase.scala

```diff
@@ -460,4 +462,11 @@ trait DeltaTableWriteSuiteBase extends AnyFunSuite with TestUtils {
         .stream()
         .anyMatch(hook => hook.getType == PostCommitHookType.CHECKPOINT) === isReadyForCheckpoint)
   }
+
```

Comment:
please document that all child suites should use this instead of `.commit` directly, and explain why this is!! also maybe put it at the top of the file in all caps (in DeltaTableWriteSuite)